config/v1/types_network.go

```diff
@@ -67,6 +72,10 @@ type NetworkStatus struct {
 
 	// ClusterNetworkMTU is the MTU for inter-pod networking.
 	ClusterNetworkMTU int `json:"clusterNetworkMTU,omitempty"`
+
+	// ServiceNodePortRange is the port range configured to use by services
+	// of type NodePort
+	ServiceNodePortRange string `json:"serviceNodePortRange,omitempty"`
```

Comment:
Do we need to expose this in Status? Do we support changing it?

The kube-apiserver-operator will need to update this when it's rolled out the configuration.

Reply:
Isn't the Status something the `clusterconfig_controller` will set? Since this is the config parameter?

Reply:
No, because the configuration is applied by the k-a-o to the control plane. It is responsible for writing the current status of the cluster (e.g. the current port range).

The CNO doesn't react to this parameter.

Reply:
In the interest of reflecting the currently configured range accurately in the `Status`, upon reconciliation by k-a-o, the k-a-o will set the port range being used in the `Status` field.

Reply:
I don't think we have cases where an operator reports status in a foreign resource. There is no mechanism to do so. It's not even clear what the semantics of this status should be: the API servers can have mixed port range during rollout.

Reply:
Or, since the status is not meaningful, we can just drop this field. I can't think of any downstream consumers of this value.

Reply:
we have hundreds of users provided values in our config and how many do we reflect into status? Doesn't seem to be a common pattern, especially when it crosses operators like here.

Reply:
If there is consensus, that the status is of limited value here, I can take it out.